src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -298,7 +298,7 @@ public boolean isInGracePeriod() {
         Date now = new Date();
         Date gracedEnd = new Date(endTime.getTime() + gracePeriod * 60000L);
 
-        return now.after(endTime) && now.before(gracedEnd);
+        return (now.after(endTime) || now.equals(endTime)) && now.before(gracedEnd);
```

Comment:
I also considered `!now.before(endTime) && now.before(gracedEnd)`, but that might be slightly less readable.